6-register/1-register.md

```diff
@@ -0,0 +1,68 @@
+# Register a Rococo Parachain
+
+Rococo is Parity's official test network for Cumulus-based parachains. The purpose of this document is to guide Cumulus
+parachain developers through the steps needed to register their parachain with the Rococo test network. This page is an
+_addendum_ to the main workshop. Before you attempt to register your parachain on a public testnet like Rococo, we
+expect that you have gone through the entire workshop and been in contact with us through our
+[#Rococo Element chat room](https://app.element.io/#/room/!WuksvCDImqYSxvNmua:matrix.parity.io?via=matrix.org)
+
+## Write Your Parachain
+
+You covered this material in the [Parachain Template Overview](../5-develop/1-template-overview.md) and
+[Template Pallet](../5-develop/3-template-pallet.md) sections.
+
+## Test Your Parachain
+
+You covered this material in the [Sending Messages](../5-develop/4-sending-messages.md) and
+[Receiving Messages](../5-develop/5-receiving-messages.md) sections.
+
+## Request ROC Tokens
+
+The symbol for the Rococo test network's native currency is ROC. You will need some ROC in order to register your Rococo
+parachain. There is a deposit required to register a parachain (currently 1,000 ROC) and you will also need some
+additional ROC in order pay transaction fees. Please use our
+[#Rococo Element chat room](https://app.element.io/#/room/!WuksvCDImqYSxvNmua:matrix.parity.io?via=matrix.org) to
+request ROC tokens.
+
+## Launch Rococo Validators
+
+In order to register a parachain with the Rococo test network, we require you to support the central Rococo relay chain
+by running at least two Rococo validator nodes. Refer to the first chapter to learn how to
+[build a Rococo validator node](../1-prep/1-compiling.md#building-a-relay-chain-node). To launch your Rococo validator
```

Comment:
@danforbes, @JoshOrndorff I just noticed that we need to specify that for Rococo, they need to build up to the latest commit and not the commit used in the workshop.

Reply:
So this paragraph should say that they need to test locally using the latest commit and only follow the tutorial steps to understand how to do it.

Reply:
If they use an older version of Polkadot, won't their client just apply the runtime upgrades as it syncs the chain? I thought that's the whole point of forkless upgrades.

Reply:
Yes, that is true but it will be using wasmi and the validator will be slow. I'm also not sure if that stands when you pass the flag `--wasm-execution=Compiled`. This may not be important when we use wasmtime because the speed will be much faster.

Reply:
Check out https://github.com/substrate-developer-hub/cumulus-workshop/pull/26

Reply:
Thanks for looking after this, guys :pray: